libmc/_client.pyx

```diff
@@ -279,8 +279,8 @@ cdef bytes _encode_value(object val, int comp_threshold, flags_t *flags):
             try:
                 enc_val = pickle.dumps(val, 2)
                 flags[0] = _FLAG_PICKLE
-            except:
-                pass
+            except Exception as e:
+                warnings.warn("[libmc] marshal & pickle both failed. type:%r err:%r" % (type_, e))
```

Comment:
This may lead to a log flood if the unpackable object is encoded again and again.

Reply:
@mckelvin https://github.com/douban/libmc/pull/66#issue-288795263 as i explain，it only output at the first time for each msg in warnings.warn(msg)。https://gist.github.com/MOON-CLJ/0f06a1e57d872c6f5f0f9e368326d406 this gist try to make it more clear。

so i think it's enough to avoid log flood in most case。